graphql_compiler/__init__.py

```diff
@@ -198,3 +199,74 @@ def get_graphql_schema_from_orientdb_schema_data(schema_data, class_to_field_typ
     schema_graph = get_orientdb_schema_graph(schema_data, [])
     return get_graphql_schema_from_schema_graph(schema_graph, class_to_field_type_overrides,
                                                 hidden_classes)
+
+
+def graphql_to_cypher_redisgraph(schema, graphql_query, parameters, type_equivalence_hints=None):
```

Comment:
Thoughts on making the name use `redisgraph_cypher` order instead of `cypher_redisgraph`? In my mind, `RedisGraph` is a modifier for `Cypher`, not the other way round.